src/aat/resources/features/F-026/F-026_ResponseData.td.json

```diff
@@ -0,0 +1,19 @@
+{
+  "_guid_": "F-026_ResponseData",
```

Comment:
id not correct. needs to be more specific to tell what kind of object this is specifying. And I guess the same or very similar objects are also used under other feature folders. wee need a shelf design at top level of data store FTAs and use objects from there as necessary.